code/drasil-lang/lib/Language/Drasil/Development/Sentence.hs

```diff
@@ -20,6 +20,10 @@ import qualified Language.Drasil.NounPhrase as NP
 import Drasil.Database.UID (HasUID(..))
 
 -- | Get short form (if it exists), else get term of an 'Idea'.
+-- Uses the UID of the 'Idea' in a 'Ch' Sentence constructor to get the short
```

Comment:
Nice improvement, and operationally correct.

@balacij you ought to be the next one editing this (later), to insert a more semantic description -- i.e. what exactly *is* a 'short form', and why is it acceptable to return the full term if there is no short form. [I do think such an explanation exists!]